src/js/base/ripe.js

```diff
@@ -1972,12 +1972,12 @@ ripe.Ripe.prototype._buildProfiles = function(engraving, profiles, context, sep
     // parses the provided engraving string so that it's possible to iterate
     // over the multiple structured values of it, then adds these same values
     // to the base values passed as argument to the method
-    const engravingProfiles = engraving ? this.owner.parseEngraving(engraving).values : [];

```

Comment:
oops

Reply:
I thinks its correct, I only removed the owner part, which is not necessary now